SortVisualization/SortVisualization/Protocols/ArrayTypeSortProtocol.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  ArrayTypeSortProtocol.swift
+//  SortVisualization
+//
+//  Created by Oleg Yankiwskyi on 5/7/18.
+//  Copyright © 2018 Oleg Yankiwskyi. All rights reserved.
+//
+
+import Foundation
+
+protocol ArrayTypeSortProtocol {
+    func at(index: Int) -> TypeSortEntity
```

Comment:
Please read about `subscript` ([]) overloading. 
It's more common to use it (I mean `model[index]`)